src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Rectangle implements Figure {
```

Comment:
The class 'Rectangle' should be declared as public to match the file name and to be accessible from other packages.